webapp/src/debugger.tsx

```diff
@@ -263,7 +265,7 @@ export class DebuggerToolbar extends data.Component<DebuggerToolbarProps, Debugg
                 <div className={`ui compact borderless menu icon mini`}>
                     <div className={`ui item link dbg-btn dbg-handle`} key={'toolbarhandle'}
                         onMouseDown={this.toolbarHandleDown.bind(this)}>
-                        <sui.Icon key='iconkey' icon={`icon ellipsis vertical`} />
+                        <sui.Icon key='iconkey' icon={`xicon bug`} />
```

Comment:
This is for the handle?? Why you changing that.

Reply:
not clear that the menu is related to debugging. it's just a bunch of arrows.